recipes-bsp/u-boot/u-boot-common.inc

```diff
@@ -0,0 +1,22 @@
+HOMEPAGE = "http://www.denx.de/wiki/U-Boot/WebHome"
+DESCRIPTION = "U-Boot, a boot loader for Embedded boards based on PowerPC, \
+ARM, MIPS and several other processors, which can be installed in a boot \
+ROM and used to initialize and test the hardware or to download and run \
+application code."
+SECTION = "bootloaders"
+DEPENDS += "flex-native bison-native"
+
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://Licenses/README;md5=30503fd321432fc713238f582193b78e"
+PE = "1"
+
+# We use the revision in order to avoid having to fetch it from the
+# repo during parse
+SRCREV = "36fec02b1f90b92cf51ec531564f9284eae27ab4"
```

Comment:
@Cezarus27 we should create new recipe for U-Boot 2020.04 and use those include by adding require (https://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html#require-inclusion) to `u-boot_2020.04.bb`

Reply:
@TomaszAIR Ok, it is fixed.